Rakefile

```diff
@@ -12,76 +12,61 @@ end
 
 task :test => [:check_test]
 
-task :generate_host_config => [:check_acceptance]do |t, args|
+task :generate_host_config do |t, args| #=> [:check_acceptance]do |t, args|
 
   target = ENV["TEST_TARGET"] || 'centos7-64'
-  generate = "bundle exec beaker-hostgenerator"
+  generate = "beaker-hostgenerator"
   generate += " #{target}"
   generate += " > acceptance/hosts.cfg"
   sh generate
   sh "cat acceptance/hosts.cfg"
 end
 
-desc 'Run acceptance tests for Rototiller'
-task :acceptance => [:check_acceptance, :generate_host_config]do |t, args|
-
-  config = ENV["BEAKER_CONFIG"]
-
-  preserve_hosts = ENV["BEAKER_PRESERVEHOSTS"]
-  keyfile = ENV["BEAKER_KEYFILE"]
-  load_path = ENV["BEAKER_LOADPATH"]
-  pre_suite = ENV["BEAKER_PRESUITE"]
-  test_suite = ENV["BEAKER_TESTSUITE"]
-
-  beaker = "bundle exec beaker "
-  beaker += " --debug"
-  beaker += " --preserve-hosts #{preserve_hosts}" if preserve_hosts != ''
-  beaker += " --hosts #{config}" if config != ''
-  beaker += " --keyfile #{keyfile}" if keyfile != ''
-  beaker += " --load-path #{load_path}" if load_path != ''
-  beaker += " --pre-suite #{pre_suite}" if pre_suite != ''
-  beaker += " --tests #{test_suite}" if test_suite != ''
-  sh beaker
-end
-
-Rototiller::Task::RototillerTask.define_task :check_acceptance do |t|
+Rototiller::Task::RototillerTask.define_task :acceptance => [:generate_host_config] do |t|
   # with a hash
   t.add_env({:name => 'TEST_TARGET',:default => 'centos7-64', :message => 'The argument to pass to beaker-hostgenerator'})
 
   # with new block syntax
-  t.add_env do |env|
-    env.name = 'BEAKER_CONFIG'
-    env.default = 'acceptance/hosts.cfg'
-    env.message = 'The configuration file that Beaker will use'
+  t.add_flag do |flag|
+    flag.name = '--hosts'
+    flag.default = 'acceptance/hosts.cfg'
+    flag.message = 'The configuration file that Beaker will use'
+    flag.override_env = 'BEAKER_HOSTS'
   end
-  t.add_env do |env|
-    env.name = 'BEAKER_PRESERVEHOSTS'
-    env.default = 'never'
-    env.message = 'The beaker setting to preserve a provisioned host'
+  t.add_flag do |flag|
+    flag.name = '--preserve-hosts'
+    flag.default = 'never'
```

Comment:
i think all the puppet tests are setup that way.  so you can go later(-ish) and login to the machine to see what's up